docs/variants/novacustom_ns5x_tgl/test-matrix.md

```diff
@@ -37,9 +37,13 @@ subjected before the release of the new binary.
 | 26.  | [Custom Boot Order][CBO]                               | CBO           | CBO001.002                           |
 | 27.  | [QubesOS support][QBS]                                 | QBS           | All                                  |
 | 28.  | [Fedora support][FED]                                  | FED           | All                                  |
-| 29.  | [Platform suspend and resume][SUSP]                    | SUSP          | SUSP001.001, SUSP002.001, SUSP003.001 |
-| 30.  | [Sign of life][SOL]                                    | SOL           | All                                  |
-| 31.  | [BIOS menu function keys][BMF]                         | BMF           | All                                  |
+| 29.  | [Platform suspend and resume][SUSP]                    | SUSP          | Without SUSP004.001 and SUSP006.001  |
+| 30.  | [Boot blocking][BBB]                                   | BBB           | All                                  |
+| 31.  | [Reset to defaults][RTD]                               | RTD           | All                                  |
+| 32.  | [Suspend mechanism switchcing (S0ix/S3)][SMS]           | SMS           | All                                 |
```

Comment:
```suggestion
| 32.  | [Suspend mechanism switchcing (S0ix/S3)][SMS]          | SMS           | All                                 |
```

Reply:
Fixed in 4856d86caf7b885bf1645c10375a7d999fd4ab71